packages/builder/src/pages/builder/portal/users/users/index.svelte

```diff
@@ -110,11 +112,18 @@
           })
         }
       })
+      user.tenantOwnerEmail = res.tenantOwner?.email
```

Comment:
We should just load the tenant info separately here, and combine that with the user list to generate our data to pass into the table. This may also have outgrown being a reactive statement, so it might be a good idea to pull this into a dedicated function now and pass in `$fetch.rows`, `$groups` and the tenant owner (and any other dependencies I've missed).